### PR TITLE
downloadInProgress should be true when download starts

### LINKED ISSUE #8: SingleDownload property: downloadInProgress is always false
When downloading a file with
```
import Ubuntu.DownloadManager 1.2

SingleDownload {
    id: single
}
```
property `single.downloadInProgress` is always false

### PR DESCRIPTION
**Pull request by cibersheep**
Fixes #8

### CODE REVIEW

**Comment by cibersheep** [comment]
@UniversalSuperBox are you ble to check this <3?

**Review by mariogrip — APPROVED** [review]
LGTM :+1:

File: src/downloads/qml/single_download.cpp [hunk 1]
```diff
@@ -367,6 +367,7 @@ void
 SingleDownload::onStarted(bool wasStarted)
 {
     m_downloading = true;
+    m_downloadInProgress = true;
```
Comment (mariogrip) [inline src/downloads/qml/single_download.cpp:370]: Thinking about it, we should also set = false once download is done

**Review by mariogrip — CHANGES_REQUESTED** [review]

File: src/downloads/qml/single_download.cpp (hunk 1 above)
Comment (cibersheep) [reply src/downloads/qml/single_download.cpp:370]: I'm not sure but I think that's already implemented here:
https://github.com/ubports/ubuntu-download-manager/blob/6f2529a58be84d90f62ed99a735f41361596b8c5/src/downloads/qml/single_download.cpp#L335

**Pull request merged**